app/server/lib/OIDCConfig.ts

```diff
@@ -35,6 +35,17 @@
  *    env GRIST_OIDC_SP_IGNORE_EMAIL_VERIFIED
  *        If set to "true", the user will be allowed to login even if the email is not verified by the IDP.
  *        Defaults to false.
+ *    env GRIST_OIDC_IDP_ENABLED_PROTECTIONS
+ *        A comma-separated list of protections to enable. Supported values are "PKCE", "STATE", "NONCE".
+ *        It's highly recommended that you enable STATE, and at least either PKCE or NONCE.
+ *        Defaults to "PKCE,STATE".
```

Comment:
I recommend switching the order of the two lines above, to reassure the reader that there is a good default value. Maybe even better to say 'Defaults to "PKCE,STATE", which is the recommended setting.' before adding the more detailed recommendation. Also, should mention the special "UNPROTECTED" value.